crates/matrix-sdk/tests/integration/room/joined.rs

```diff
@@ -642,3 +648,67 @@ async fn report_content() {
 
     room.report_content(event_id, Some(score), Some(reason.to_owned())).await.unwrap();
 }
+
+#[async_test]
+async fn subscribe_to_typing_notifications() {
+    let (client, server) = logged_in_client().await;
+    let typing_sequences: Arc<Mutex<Vec<Vec<OwnedUserId>>>> = Arc::new(Mutex::new(Vec::new()));
+    let asserted_typing_sequences =
+        vec![vec![user_id!("@alice:matrix.org"), user_id!("@bob:example.com")], vec![]];
+    let room_id = room_id!("!test:example.org");
+    let mut ev_builder = SyncResponseBuilder::new();
+
+    // Initial sync with our test room.
+    ev_builder.add_joined_room(JoinedRoomBuilder::new(room_id));
+    mock_sync(&server, ev_builder.build_json_sync_response(), None).await;
+    let sync_settings = SyncSettings::new().timeout(Duration::from_millis(3000));
```

Comment:
```suggestion
    let sync_settings = SyncSettings::new().timeout(Duration::from_secs(3));
```

Reply:
We are using `SyncSettings::new().timeout(Duration::from_millis(3000)); everywhere in the tests, wanted to keep the same way